### ISSUE TITLE
chp1 technical hurdles

### ISSUE THREAD

**Issue by hardin47** [OpenIntroStat/randomization-and-simulation#13]
The good news is that all of chapter 1 has been successfully merged into bookdown and can be seen at https://isrs2.netlify.com/intro-to-data.html#numericalData

The bad news is that there is a long list of technical details (some big, some small) that need to be addressed.  I've tried to lay out the problems here, but please let me know if any of this doesn't make sense.  I look forward to chatting more soon!

FYI:  I tried to complete everything at the sentence level, but that sometimes got messed up when I was inside an environment like a footnote (so I didn’t do it).

Text / content:  I used the .tex files from:  https://github.com/OpenIntroStat/randomization-and-simulation/releases but I remember you saying something about using the newer examples from the updated main text.  How to work through that???

Chapter 1:

•	Last section.  Which should it be?  

`## Revisit case study with new terminology we learned`  OR:

\section{Examining numerical data}
\label{numericalData}

•	Changed all \term to \index (the argument will be blank in rendered html!), everywhere is says:  \index{data}data
•	Table 1.1 instead of Figure 1.1
•	What does \data refer to?  I didn’t change any of them (and there are *many* to change!).
•	In the `email50` data, your text says `html` and `text` and the actual dataset has the variable coded as 0/1  (same with spam yes/no).

•	I changed \resp{none} to `none`  (for all \resp items)
•	I didn’t keep references that refer to page numbers.  Do you have a way to connect to the ideas?

\begin{enumerate}
\setlength{\itemsep}{0mm}
\item[(1)]\label{fedSpendingPovertyQuestion} Is federal spending, on average, higher or lower in counties with high rates of poverty?
\item[(2)]\label{ownershipMultiUnitQuestion} If homeownership is lower than the national average in one county, will the percent of multi-unit structures in that county likely be above or below the national average?
\item[(3)]\label{isAverageIncomeAssociatedWithSmokingBans} Which counties have a higher average income: those that enact one or more smoking bans or those that do not?
\end{enumerate}

•	What is \vref?  I treated it just like \ref .
•	What do you want to do with \begin{termBox}{\tBoxTitle{Associated or independent, not both}  (There is also a \begin{tipbox} … I skipped all of them.)
•	And similarly, what to do with \begin{caution}{association does not imply causation} such boxes?
•	Seems like include_graphics wants .jpg or .png, not .pdf …
•	I commented out \APVersion{
•	I don’t think bookdown is going to support the margin indexing.  How do you feel about losing that?
•	The “line” in the middle of every example doesn’t serve the same role as it did in the pdf, but I don’t know what else to put there.
•	I didn’t put the R code for the figures into the text.  I probably should have, but ...
•	Latex package subfig should put fig.subcap=c('(a)', '(b)'), into images, but it doesn’t show up.

**Comment by hardin47** [comment]
update 1/16/2020

I broke up chapters 1 and 2.  Both are full in markdown and on https://isrs2.netlify.com/index.html.  I also added tidy R components to both chapters.  I wrote some words and added a little bit of context, but really, there aren't very many words or very much context.  I'm not totally sure how much you think should be there, so I'm going to pause on that part until after you've looked through it.

Next up for me:  I have a game plan for my class, and it's mostly starting with chapter 3.  Which means that now I need to go through chapter 3 and put it on markdown.  :)  But i have weekly labs as well, and those will start immediately with chapters 1 & 2.    I'm fully mentally prepared to make my syllabus, so the process of doing this has been *perfect* for me!

A few technical glitches along the way:

* something odd with the output.  I can do 

glimpse(data)

but not 

head(data)

i have to do:

head(data) %>% kable()

* citations?   @Wickham14 in chapter 1

* dataframe or data frame? dataset or data set?

* more words in various sections.  i'd love to know what you think is the priority here!
